airbyte-integrations/connectors/source-okta/source_okta/custom_authenticators.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC
+from dataclasses import InitVar, dataclass
+from typing import Any, Mapping, Tuple
+
+import requests
+from airbyte_cdk.sources.declarative.auth import DeclarativeOauth2Authenticator
+from airbyte_cdk.sources.declarative.auth.declarative_authenticator import DeclarativeAuthenticator
+from airbyte_cdk.sources.declarative.types import Config
+from airbyte_cdk.sources.streams.http.auth import Oauth2Authenticator
+
+
+@dataclass
+class CustomBearerAuthenticator(DeclarativeAuthenticator):
+    """
+    Custom authenticator that uses "SSWS" instead of "Bearer" in the authorization header.
+    """
+
+    config: Config
+    parameters: InitVar[Mapping[str, Any]]
+
+    @property
+    def auth_header(self) -> str:
+        return "Authorization"
+
+    @property
+    def token(self) -> str:
+        return f"SSWS {self.config['credentials']['api_token']}"
+
+
+@dataclass
+class CustomOauth2Authenticator(DeclarativeOauth2Authenticator):
```

Comment:
Please add the unit tests for the custom components

Reply:
Added tests for custom components